src/main/groovy/life/qbic/datamodel/accounting/CostEstimate.groovy

```diff
@@ -0,0 +1,24 @@
+package life.qbic.datamodel.accounting
+
+import life.qbic.datamodel.persons.Person
+import life.qbic.datamodel.projects.ProjectInfo
+
+/**
+ * A cost estimate for a project
+ *
+ * During project planning a cost estimate is often required to list expected costs.
+ * This estimate is not intended to be legally binding. It provides an overview over the project
+ * scope and the costs associated with it.
+ *
+ * @since: 1.9.0
+ * @author: Tobias Koch
+ *
+ * * */
+class CostEstimate {
+    Date modificationDate
+    Person customer
+    ProjectInfo projectInfo
```

Comment:
As far as I understand ProjectInfo in our context is project title and description. I would suggest to separate this from the openbis ProjectInfo as used here.

Reply:
The [ProjectInfo](https://github.com/qbicsoftware/data-model-lib/blob/feature/qoffer2/src/main/java/life/qbic/datamodel/projects/ProjectInfo.java) already includes information about the project title, the secondary title, and description. Since this class represents information on project and only acts as a DTO (no logic), I would prefer to use the QBiC global definition.